lib/IAMClient.js

```diff
@@ -287,18 +285,18 @@ class VaultClient {
      * Update Quota of an account
      *
      * @param {string} accountName - account name
-     * @param {number} quota - maximum quota for the account
+     * @param {bigint} quota - maximum quota for the account
      * @param {VaultClient~requestCallback} callback - callback
      * @returns {undefined}
      */
     updateAccountQuota(accountName, quota, callback) {
         const data = {
             Action: 'UpdateAccountQuota',
             Version: '2010-05-08',
-            quotaMax: quota,
+            quotaMax: quota.toString(),
```

Comment:
this happens before validation, so may crash if `quota` is not a bigint

Reply:
Validation was moved before. Also supports simple numbers for backward compatibility (with the risk of missing places where it's still a number)